### PR TITLE
Create README.md

### LINKED ISSUE #3: Project ReadMe
Initial ReadMe giving an overview of the projects standards and setup.

To Do:
- [x] Project Standards
- [x] Setup Definition

### PR DESCRIPTION
**Pull request by tristanphip**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+### Setting Up Unity
+1. Open Unity Hub.
+2. Click "Add" and navigate to the directory where you cloned the repository.
+3. Select the project and open it.
+
+### Installing Dependencies
+Upon opening the project, Unity will automatically resolve any package dependencies via the Unity Package Manager. Ensure all packages are installed without errors.
+
+## Project Structure
```
Comment (ryanbasiltrickett) [inline README.md]: I think this project structure might needs some work

File: README.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+- **Art Assets:** Store all sprite assets in the `/Assets/Art/Sprites` directory.
+- **Audio Files:** Store all audio assets in `/Assets/Audio`. Ensure audio files are appropriately compressed for optimal performance.
+
+### Asset Import Settings
+- **Sprites:** Ensure sprites are imported with appropriate compression and pixel settings to maintain performance and visual quality.
+- **Audio:** Compress audio files and set import settings to optimize for platform performance.
+
+## Testing
+
+### Unit Testing
```
Comment (ryanbasiltrickett) [inline README.md]: Don't think we will do unit testing

File: README.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+### Creating a New Branch
+```bash
+git checkout main
+git pull
+git checkout -b feature/your-feature-name
+```
+
+### Merging Branches
+Once your work is complete and tested, open a pull request to merge your feature branch into `dev`. Ensure your branch is up to date with `dev` before requesting a merge.
```
Comment (ryanbasiltrickett) [inline README.md]: Should be main branch

**Review by ryanbasiltrickett — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (tristanphipson) [reply README.md]: What does ye suggest?

File: README.md (hunk 2 above)
Comment (tristanphipson) [reply README.md]: Facts this is just there, can remove

File: README.md (hunk 3 above)
Comment (tristanphipson) [reply README.md]: Oh yes this from git flow

**Review by ryanbasiltrickett — APPROVED** [review]
LGTM

**Pull request merged**